packages/api/.env.dev

```
@@ -7,3 +7,6 @@ PG_USER=postgres
 PG_PASSWORD=postgres
 
 JWT_SECRET=secret
+
+ETH_EDGE_URL=192.168.2.24
```

Since the containers are all on the same docker network you can just refer to them by their host name, also changing the URL part to HOST is more representative

```suggestion
ETH_EDGE_HOST=eth-edge
```